src/plugins/discover/public/application/main/components/top_nav/discover_topnav.tsx

```diff
@@ -221,7 +219,7 @@ export const DiscoverTopNav = ({
     onChangeDataView,
     textBasedLanguages: supportedTextBasedLanguages as DataViewPickerProps['textBasedLanguages'],
     adHocDataViews: adHocDataViewList,
-    savedDataViewList,
+    savedDataViews: savedDataViewList,
```

Comment:
Ok, so the new state is now the role model:

```
export interface InternalState {
  dataView: DataView | undefined;
  savedDataViews: DataViewListItem[];
  adHocDataViews: DataView[];
}
```
there is some more renaming that can be considered, mainly functions, but this would be a 🐰 🕳️ ... so the new kids on the block go first. Thx for this suggestion!